src/pages/PokemonDetail.tsx

```diff
@@ -0,0 +1,116 @@
+/** @format */
+import {
+	useContext,
+	useState,
+	useEffect,
+} from "react";
+import { PokemonContext } from "../context/PokemonContext";
+import "./PokemonDetail.css";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import { NavLink } from "react-router-dom";
+import {
+	Button,
+	CardActionArea,
+	CardActions,
+	Box,
+} from "@mui/material";
+import {
+	Ability,
+	PokemonDetail,
+} from "../context/Domain";
+
+const PokemonDetailComponent = () => {
+	const pokemonContext = useContext(
+		PokemonContext
+	);
+	const selectedPokemon =
+		pokemonContext?.selectedPokemon;
+
+	const [
+		pokemonDetail,
+		setPokemonDetail,
+	] = useState<PokemonDetail | null>();
+
+	useEffect(() => {
+		setPokemonDetail(
+			pokemonContext?.pokemonDetail
+		);
+	}, [pokemonContext?.pokemonDetail]);
+	return (
+		<div className="DetailScreen">
```

Comment:
Few other options are to use styled components or just inline styling.
I am not a big fan of mixing different ways of applying CSS, unless there is a real need for that.

Reply:
fixed